src/aind_metadata_mapper/stimulus/camstim.py

```diff
@@ -104,7 +104,13 @@ def __init__(
 
             self.session_start = sync.get_start_time(sync_data)
             self.session_end = sync.get_stop_time(sync_data)
-            self.build_stimulus_table()
+
+            pkl_data = pkl.load_pkl(self.pkl_path)
+            is_behavior = pkl.check_if_behavior(pkl_data)
+            if is_behavior:
+                self.build_behavior_table()
```

Comment:
behavior_utils is missing from this repository unless I am missing something https://github.com/AllenNeuralDynamics/aind-metadata-mapper/blob/723d5237ece5b680be0d5f57f799e96478a053e0/src/aind_metadata_mapper/stimulus/camstim.py#L20